src/js/tab-switcher.js

```diff
@@ -0,0 +1,27 @@
+import homePage from './home'
+import menuPage from './menu'
+import aboutPage from './about'
+
+function removeCurrentTab() {
+  const nodesToRemove = document.querySelector('.inner-container').childNodes;
+  nodesToRemove.forEach(node => node.remove());
+}
+
+function tabSwitcher(pageName) {
+  removeCurrentTab();
+  const innerContainer = document.querySelector('.inner-container');
+
+  switch(pageName) {
+    case 'Home':
+      innerContainer.appendChild(homePage);
+      break;
+    case 'Menu':
+      innerContainer.appendChild(menuPage);
+      break;
+    case 'About':
+      innerContainer.appendChild(aboutPage);
+      break;
+  }
+}
```

Comment:
- [ ] [REQUIRED] Each module should be able to append itself to the DOM
![image](https://user-images.githubusercontent.com/73254040/99142438-88bc9e80-261a-11eb-8f3e-1ace27fd9327.png)
[Source](https://www.theodinproject.com/courses/javascript/lessons/restaurant-page)

Reply:
Fixed the code so that each module can append itself to the DOM.